GitUI/BranchTreePanel/RemoteBranchTree.cs

```diff
@@ -31,7 +31,8 @@ protected override Nodes FillTree(IReadOnlyList<IGitRef> branches, CancellationT
             ConfigFileRemoteSettingsManager remotesManager = new(() => Module);
 
             // Create nodes for enabled remotes with branches
-            foreach (IGitRef branch in branches)
+            // If prio is set, exclude "refs/remotes/<remote>/"
```

Comment:
Don't abbreviate

Reply:
Removing this comment, not adding info here